### DISCUSSION TITLE
How do I make LSP progress background transparent?

### DISCUSSION THREAD

**Discussion by Alienover** [folke/noice.nvim#509] [Q&A]
![image](https://github.com/folke/noice.nvim/assets/12182069/04b0546c-9d2e-48dd-8981-cd1fde6c6202)
```lua
require("noice").setup({ 
  lsp = {
    progress = {
      enabled = true,
  }
})
```
Currently, the background from percentage to client name is `Black`. Wanna ask whether there's a highlight group I can update to make the background transparent?

Actually, I can get the LSP progress with transparent background by using [fidget.nvim](https://github.com/j-hui/fidget.nvim). But I am curious about whether Noice can do that. If so, I don't have to install one more plugin.
![image](https://github.com/folke/noice.nvim/assets/12182069/f578b5af-e7d6-40fb-b800-8949f4172804)

FYI, my color scheme is `catppuccin-mocha`.

**Comment by alex-popov-tech** [comment]
for anyone looking for an answer, some reference could be found here https://github.com/folke/noice.nvim/issues/226
